VideoLocker/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.edx.mobile.util;
+
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.module.db.DbStructure;
+import org.edx.mobile.module.prefs.PrefManager;
+
+import java.io.File;
+import java.util.Collections;
+
+public abstract class SecurityUtil {
```

Comment:
Also, could we have some javadocs on the scope of this class, and the `clearUserData()` method?